src/DynamoCore/Nodes/ZeroTouch/DSVarArgFunction.cs

```diff
@@ -117,16 +117,9 @@ protected override void BuildOutputAst(NodeModel model, List<AssociativeNode> in
             if (!model.IsPartiallyApplied)
             {
                 var paramCount = Definition.Parameters.Count();
-                var packId = "__var_arg_pack_" + model.GUID;
-                resultAst.Add(
-                    AstFactory.BuildAssignment(
-                        AstFactory.BuildIdentifier(packId),
-                        AstFactory.BuildExprList(inputAstNodes.Skip(paramCount - 1).ToList())));
-
-                inputAstNodes =
-                    inputAstNodes.Take(paramCount - 1)
-                        .Concat(new[] { AstFactory.BuildIdentifier(packId) })
-                        .ToList();
```

Comment:
Hi @ke-yu, if you can help adding comments to this as discussed, it'll make the code that much clearer. Thanks!